core/src/main/java/nu/yona/server/messaging/service/FirebaseService.java

```diff
@@ -39,6 +40,7 @@
 @Service
 public class FirebaseService
 {
+	private final static String FIREBASE_ID_NOT_REGISTERED = "registration-token-not-registered";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)